SmartDeviceLinkTests/DevAPISpecs/SDLLockScreenManagerSpec.m

```diff
@@ -228,66 +244,77 @@ @interface SDLLockScreenManager ()
         it(@"should should not modify a custom lockscreen", ^{
             testsConfig.showDeviceLogo = YES;
             UIViewController *customLockScreen = [[UIViewController alloc] init];
-            fakeViewControllerPresenter.lockViewController = customLockScreen;
+            OCMStub([fakeViewControllerPresenter lockViewController]).andReturn(customLockScreen);
             testManager = [[SDLLockScreenManager alloc] initWithConfiguration:testsConfig notificationDispatcher:dispatcherMock presenter:fakeViewControllerPresenter];
 
             [[NSNotificationCenter defaultCenter] postNotificationName:SDLDidReceiveLockScreenIcon object:nil userInfo:@{ SDLNotificationUserInfoObject: testIcon }];
 
-            expect(fakeViewControllerPresenter.lockViewController).toEventually(equal(customLockScreen));
+            expect(((SDLFakeViewControllerPresenter *)fakeViewControllerPresenter).lockViewController).toEventually(equal(customLockScreen));
         });
     });
 
     context(@"with a custom color configuration", ^{
+        __block id fakeViewControllerPresenter = nil;
+        __block NSNotificationCenter *dispatcherMock = nil;
```

Comment:
Why are you re-declaring these everywhere instead of the way that it was?

Reply:
Fixed. Refactored to set each property back to `nil` before each test.